appservice/src/main/java/nu/yona/server/device/rest/DeviceController.java

```diff
@@ -211,6 +227,20 @@ private void logLongAppActivityBatch(MessageType messageType, UUID userId, AppAc
 		}
 	}
 
+	public static Link getPostOpenAppEventLink(UUID userId, UUID deviceId)
+	{
+		try
+		{
+			ControllerLinkBuilder linkBuilder = linkTo(
+					methodOn(DeviceController.class).postOpenAppEvent(Optional.empty(), userId, deviceId, null));
+			return linkBuilder.withRel("postOpenAppEvent");
+		}
+		catch (SecurityException e)
```

Comment:
Why would this ever give a `SecurityException`?

Reply:
No idea. It came this way from ``UserController``, but it isn't necessary. Removed it.